tests/test_verifylib.py

```diff
@@ -1023,7 +1031,19 @@ def test_verify_passing_double_signed_layout(self):
             [self.alice_path, self.bob_path]
         )
         in_toto_verify(layout, layout_key_dict)
-
+    
+    def test_verify_failing_layout_current_datetime(self):
+        """Test pass verification with current date expiry layout."""
+        with patch("""in_toto.verifylib.datetime.datetime""") as mock_date:
+            with patch("""in_toto.verifylib.iso8601""") as mock_date2:
+                mock_date.now.return_value= datetime(2025, 1, 2, 3, 4, 6)
+                mock_date2.parse_date.return_value = datetime(2025, 1, 2, 3, 4, 6)
+                assert in_toto.verifylib.datetime.datetime.now() == in_toto.verifylib.iso8601.parse_date(datetime.now())
+                layout = Metablock.load(self.layout_current_datetime_expiry_path)
+                layout_key_dict = import_publickeys_from_file([self.alice_path])
+                with self.assertRaises(LayoutExpiredError):
+                    in_toto_verify(layout, layout_key_dict)
```

Comment:
```
-> if expire_datetime <= datetime.datetime.now(tz.tzutc()):
(Pdb) expire_datetime
datetime.datetime(2025, 1, 2, 3, 4, 6)
(Pdb) datetime.datetime.now()
datetime.datetime(2025, 1, 2, 3, 4, 6)
(Pdb)
```

A breakpoint in verifylib indicates this works fine **logically**, and the test indeed passes. However, I think the setup can be better.